auth.py

```diff
@@ -22,13 +24,19 @@ def pwhash(password):
     return pwh
 
 def add_user(pwdb, username, password):
+    salty=salt()
     if username not in pwdb:
-        pwdb[username] = password
+        pwdb[username] = ("".join(salty),pwhash("".join(salty)+password))
+def salt():
+    chars=list(string.ascii_lowercase)
+    random.shuffle(chars)
+    return chars[0:5]
```

Comment:
you could `join` here, so you can get a nice string from this function to avoid having to do it outside